worker/worker.go

```diff
@@ -24,14 +24,9 @@ type Status struct {
 	LastFinished time.Time
 	// Idle stands for whether worker is idle, false if syncing
 	Idle bool
-	// Stdout records outputs to stdout of each command execution
-	// TODO: This slice may grow quite large as time goes by
-	// Candidate solutions:
-	// - use compression method like gzip/zlib [logs tend to have high compression rate, but just workaround]
-	// - keep max-length and remove exceeded items [frequent memory operation, stop-the-world GC]
-	// Anyway, a new type needs to be implemented/imported with Put(string) and GetAll() []string method
+	// Last stdout(s) for admin. Internal implementation may vary to provide it in Status()
 	Stdout []string
```

Comment:
Should we really hold stdout and stderr in memory with slices of strings. What if the shell scripts has huge output. Maybe we should find some way to streaming the output into ELK and let them handling it for us in the future, :p